src/helpers/getResultsByResource.ts

```diff
@@ -0,0 +1,33 @@
+import omit from 'lodash/omit';
+import uniq from 'lodash/uniq';
+import { ChecksResults, FailedRule, ResourceResult } from '../types';
+
+export const getResultsByResource = (
```

Comment:
This function is quite difficult to understand. Have a look at this: https://refactoring.guru/extract-method.
Not blocking.